portfolio/src/main/webapp/script.js

```diff
@@ -134,6 +134,12 @@ function createCommentNode(comment) {
   dateElement.classList.add("comment-name-date");
   dateElement.appendChild(dateNode);
   comDiv.appendChild(dateElement);
+  
+  const scoreNode = document.createTextNode(`Sentiment analysis score: ${comment.score !== undefined && comment.score !== 2 ? comment.score : "N/A"}`);
```

Comment:
Nitpick: Instead of the pure score, you could give a textual description ("your comment has a negative sentiment") or an emoji :smiley:

Reply:
done!